tests/ClientTest.php

```diff
@@ -40,7 +40,7 @@ public function testGetPublicationSummaries(): void
         $this->assertEquals(2, $result[1]->id);
 
         $request = $this->getLastRequestParameters();
-        $this->assertIsString($request['LastModified']);
+        $this->assertIsString($request['LastModified'] ?? null);
```

Comment:
als lastModified null is , returned hij null ?

Reply:
Deze change was vooral om PHPStan tevreden te houden. Het verandert het resultaat van de assertion niet, maar in het geval dat $request geen LastModified key zou hebben, zou er nog een "Undefined array key LastModified" warning bovenop komen. Daar is PHPStan niet zo blij mee.